examples/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## [Cloropleth](https://github.com/procyon-rs/vega_lite_3.rs/blob/master/examples/cloropleth_unemployment.rs)
 
-```
+```rust
```

Comment:
those are actually shell scripts and not rust samples

Reply:
true